src/dom.js

```diff
@@ -204,6 +204,49 @@ export function closestByTag(element, tagName) {
   });
 }
 
+/**
+ * Finds the closest element with the specified selector from this element
+ * @param {!Element} element
+ * @param {string} selector
+ * @return {?Element} closest ancestor if found.
+ */
+export function closestBySelector(element, selector) {
+  if (element.closest) {
+    return element.closest(selector);
+  }
+
+  return closest(element, el => {
+    return matches(el, selector);
```

Comment:
My point is this: if we decide we do NOT need a fallback, let's eliminate completely. If we decided that we DO need fallback - it should be inside the `matches` method. So, let's confirm for sure. If this is ONLY b/c of IE8 - let's remove the fallback.